packages/node/src/utils/commonjs.ts

```diff
@@ -1,4 +1,8 @@
 /** Detect CommonJS. */
 export function isCjs(): boolean {
-  return typeof require !== 'undefined';
+  try {
```

Comment:
IMHO if we do the try catch, we can omit the `&& module` check xD as we should be safe already!

Reply:
suuper nit xD